.circleci/config.yml

```diff
@@ -24,6 +24,14 @@ commands:
           command: |
             echo "export SENTRY_RELEASE=${SENTRY_PROJECT}@${CIRCLE_SHA1}" >> $BASH_ENV
 
+  dump-env-vars:
+    steps:
+      - run: ./env_dump.sh
+      - run:
+          name: Prepare private env variables
+          command: |
+            cat .env.local
```

Comment:
Don't think it's a good behaviour to show the env content. As some of the env variables are confidential information, so we'd keep it secret as much as possible.